assets/scripts/dialogs/Analytics/SegmentCapacity.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
```

Comment:
I can't tell where this component is being used at all. Tell me more about this?

Reply:
This component was added to let users manually update the value on a particular segment, but the UI has since moved in a different direction ✂️